foundation-extensions/profile/README.md

```diff
@@ -30,7 +30,7 @@ To add the Profile extension to your app:
    ```
 {% endtab %}
 
-{% tab title="iOS — Swift" %}
+{% tab title="iOS (AEP 3.x)" %}
```

Comment:
This change looks ok for now, we need to revisit this and have swift first, followed by obj-c examples for both 3.x and 2.x